libr/core/linux_heap_glibc.c

```diff
@@ -1114,8 +1114,8 @@ static void GH(print_heap_segment)(RCore *core, MallocState *main_arena,
 	char *top_title, *top_data, *node_title, *node_data;
 	bool first_node = true;
 
-	top_data = r_str_newf ("");
-	top_title = r_str_newf ("");
+	top_data = r_str_new ("");
```

Comment:
or just strdup

Reply:
I preserved it for the consistency with the rest of the code. To be honest, that whole file should be refactored, imho.